### PR TITLE
CB-8853 AWS availibility zone json files contain conflicting data

### PR DESCRIPTION
**Pull request by bbihari**
See detailed description in the commit message.

### CODE REVIEW

File: core/src/test/java/com/sequenceiq/cloudbreak/logger/MDCBuilderTest.java [hunk 1]
```diff
@@ -12,13 +12,15 @@
 
     @Test
     public void buildSimpleContext() {
+        MDC.clear();
```
Comment (Bajzathd) [inline core/src/test/java/com/sequenceiq/cloudbreak/logger/MDCBuilderTest.java:15]: These changes seem unrelated to the ticket. Please write details into the commit message why it was needed. 
Also, it could be moved to a `@Before` method.

Comment (bbihari) [reply core/src/test/java/com/sequenceiq/cloudbreak/logger/MDCBuilderTest.java:15]: The test failed on Linux without clearing the MDC context.

Comment (pdarvasi) [reply core/src/test/java/com/sequenceiq/cloudbreak/logger/MDCBuilderTest.java:15]: Could you pls use `@Before` then, @bbihari ?

Comment (bbihari) [reply core/src/test/java/com/sequenceiq/cloudbreak/logger/MDCBuilderTest.java:15]: I will remove this file from this pull and create a separate pull request

**Review by Bajzathd — APPROVED** [review]

**Review by pdarvasi — APPROVED** [review]
